expression/builtin_time_test.go

```diff
@@ -1248,3 +1249,27 @@ func (s *testEvaluatorSuite) TestTimeToSec(c *C) {
 		c.Assert(result.GetInt64(), Equals, test.expect)
 	}
 }
+
+func (s *testEvaluatorSuite) TestSecToTime(c *C) {
+	// Test cases from https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_sec-to-time
+	tests := []struct {
+		param  interface{}
+		expect string
+	}{
+		{2378, "00:39:38"},
+		{3864000, "838:59:59"},
+		{-3864000, "-838:59:59"},
+		{86401.4, "24:00:01"},
+		{86401.5, "24:00:02"},
```

Comment:
Please add cases for `nil` and valid string like "123" and  invalid string like "abc".